src/Service.GraphQLBuilder/Sql/SchemaConverter.cs

```diff
@@ -424,16 +424,21 @@ private static FieldDefinitionNode GenerateFieldForColumn(Entity configEntity, s
             }
 
             string exposedColumnName = columnName;
-            if (configEntity.Mappings is not null && configEntity.Mappings.TryGetValue(key: columnName, out string? columnAlias))
+            FieldMetadata? fieldMetadata = null;
+            if (configEntity.Fields is not null)
```

Comment:
We should continue to check for configEntity.Mappings is not null. Both cant be true but either could be.